docs/content/language-cheatsheet.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Language Cheatsheet
+navtitle: Language Cheatsheet
+kind: documentation
+weight: 5
+toc: true
+---
+
+## Assignment and Equality
+
+**Outside Rules**
+
+```ruby
+# assign variable allowed to the set "hooli.com" and "initech.com"
+allowed = {"hooli.com", "initech.com"}
```

Comment:
NIT: `:=`

Reply:
This is _outside of a rule_ so it's correct in this case. I think the inside/outside distinction is confusing. I'll try to refactor the example.